email_slicer_using_Cli_handler.py

```diff
@@ -0,0 +1,32 @@
+class Email_slicer:
```

Comment:
We shall not mix "_" naming convention with snake case. In Python the convention is to use snake case for class name and underscore for method names, so the class name should be `EmailSlicer`